platforms/hyperledger-besu/configuration/roles/create/certificates/ambassador/tasks/main.yaml

```diff
@@ -6,6 +6,99 @@
 
 # This role calls for ambassador certificate creation for each node.
 ---
+##Checks if ClusterIssuer is installed 
```

Comment:
Please rename all task comments uniformly to something like this
`# task_description_starting with a capital letter` (Single #) (space between # and task description)

Reply:
fixed in pr #2187 as unwanted files got added while squashing